packages/bot-analytics/test/unit/unit.test.ts

```diff
@@ -0,0 +1 @@
+describe("test", () => {});
```

Comment:
We should definitely add some unit tests

Reply:
Some of the integration test are actually unit test, I will transfer them to the correct directory